### DISCUSSION TITLE
Is everyone getting a lot of import errors?

### DISCUSSION THREAD

**Discussion by robotlightsyou** [Slow-Scallion4183/Adventure-Colussus#61] [Q&A]
Hi all,

Since we switched to the `from adventure_colussus import *` method I've been getting a lot of import errors. This didn't bother me because the program ran and I was mostly using print debugging. I'm moving past that and trying to do real debugging in VS Code, but I can't get it to run because of the import errors. I'm also having trouble importing pytest and mock. Mock in particular is confusing since it's a builtin, I think. Any thoughts? I think this is a vs code thing, because when I open the files in neovim I don't get the errors.

Further details: I'm running a pipenv shell in the directory above `Adventure_Colussus` so everything in the repo is contained. I've run `python3 -m pip install -r requirements.txt`*, and when I check `help("modules") in the interpreter all the dependencies show up.

*I've had issues in the past with different versions of python installing to different locations, so I've adopted this whenever pip installing to try and make sure everything winds up on PATH.

**Comment by wireboy5** [comment]
The root of the git repo needs to be added to the python-path.
What is happening is the tests are looking for the adventure_colussus module in the path, but are not finding it. Next, they are looking for it in the tests/ directory, but can not find it. This means that a shim may be needed to add adventure_colussus to the python path before executing the tests. I am currently working on the issue.
